src/Control/Email/Email.php

```diff
@@ -352,7 +367,7 @@ public function getReturnPath()
      */
     public function setReturnPath($address)
     {
-        $this->getSwiftMessage()->setReturnPath($address);
+        $this->getSwiftMessage()->setReturnPath(trim($address));
```

Comment:
Looks like this one should be updated to use `$address = $this->sanitiseAddress($address);` like the other methods?

Reply:
Updated